build/mark-new-version.sh

```diff
@@ -92,6 +92,10 @@ fi
 
 echo "+++ Running ./versionize-docs"
 ${KUBE_ROOT}/build/versionize-docs.sh ${NEW_VERSION}
+
+echo "+++ Updating swagger"
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
```

Comment:
I think it will be better to do this at the end of versionize-docs.
That is the script that is changing types.go and hence should regenerate swagger.

Replace hack/update-swagger-spec.sh by /hack/update-generated-swagger-docs.sh there.